### ISSUE TITLE
constant sync button for selective sync project

### ISSUE THREAD

**Issue by PatrikMizera** [MerginMaps/mobile#2942]
If you use our project for selective sync, there will be a constant sync button, until you disable the selective sync in QGIS.

1. clone on production tc08_selective_sync
2. download it on mobile 
3. you will see that the project has a sync button available 
4. you can press it as many times as you want, but the sync button will remain there, even though there are no changes happening 
5. only after you disable the selective sync in QGIS, and then sync, the sync button will disappear 

https://github.com/MerginMaps/mobile/assets/94351263/571ab52c-4089-4790-883e-33e69e4f1b7b

**Comment by PeterPetrik** [comment]
is this regression or new bug in RC2.5.0?

**Comment by PatrikMizera** [comment]
it is regression, but I believe it was not caught previously

**Comment by PeterPetrik** [comment]
> it is regression, but I believe it was not caught previously

so was it present in 2.4.1 or not?

**Comment by PatrikMizera** [comment]
yes, I was able to reproduce it on 2.4.1 as well

**Comment by jozef-budac** [comment]
Still present even in 2024.1.0 RC3 candidate

**Comment by tomasMizera** [comment]
Duplicate of https://github.com/MerginMaps/mobile/issues/3129
